midscribe-core/src/test/java/com/evolveum/midscribe/TemplateUtilsTest.java

```diff
@@ -34,7 +35,7 @@ public void testDescribeCapability() throws Exception {
         variables.put("capabilities", capabilities);
 
         Properties props = new Properties();
-        props.setProperty(GeneratorProperties.VELOCITY_START_TEMPLATE, "/template/capabilities.vm");
+        props.setProperty(GeneratorProperties.VELOCITY_START_TEMPLATE, "/template/capabilities-native.vm");
```

Comment:
why only native?

Reply:
The reason for that is that i broke the 1 template into 2 and after that i did not had enough time to recreate tests. Thats why only native template.